src/components/Form.tsx

```diff
@@ -1,10 +1,32 @@
-const Form = (): JSX.Element => {
+import { useForm, SubmitHandler } from "react-hook-form";
+
+type FormData = {
```

Comment:
```suggestion
export type FormData = {
```
このForDataをexportして`App.tsx`でimportして使用すれば、`App.tsx`側でのtype Itemを定義しなくて済みます。
この提案は同じtypeを使いまわしているのでtypescript的にも正しい使い方になります。